ocs_ci/helpers/network_policy_helpers.py

```
@@ -0,0 +1,400 @@
+        pod_name (str): Pod name.
+        namespace (str): Pod namespace. Defaults to cluster_namespace.
+        hostname (str): Hostname to resolve.
+            Defaults to kubernetes.default.svc.cluster.local.
+
+    Raises:
+        AssertionError: If DNS resolution fails.
+    """
+    namespace = namespace or config.ENV_DATA["cluster_namespace"]
+    hostname = hostname or "kubernetes.default.svc.cluster.local"
+    pod_obj = get_pod_obj(pod_name, namespace=namespace)
+    cmd = f'python3 -c "import socket; ' f"print(socket.gethostbyname('{hostname}'))\""
```

Ruff (0.16.0)

[warning] Implicitly concatenated string literals on one line

(ISC001)